examples/replit-login-loop.js

```diff
@@ -0,0 +1,77 @@
+const { example } = require("../src/helpers");
+
+const selectors = {
```

Comment:
No need to do this manually. You can create Playwright tests with codegen. I have added [this video](https://www.youtube.com/watch?v=LM4yqrOzmFE) on how it works to [TT-1090](https://linear.app/replay/issue/TT-1090/crash-fix-automate-the-record-then-replay-until-it-crashes-loop).

Reply:
Update: I see what you did. I guess, you copy-and-pasted the QA Wolf script 👀